next.config.mjs

```diff
@@ -1,6 +1,21 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-    output: "standalone"
```

Comment:
also can you confirm if the docker build still works, im pretty sure you need this for the build

Reply:
That was a mistake, added back standalone output to the config.